lgc/include/lgc/patch/PatchBufferOp.h

```diff
@@ -109,7 +106,7 @@ class BufferOpLowering {
   llvm::IRBuilder<> m_builder;
 
   PipelineState &m_pipelineState;
-  llvm::DivergenceInfo &m_divergenceInfo;
+  UniformityInfo &m_uniformityInfo;
```

Comment:
Probably still need `llvm::` here - does it cause a build failure?

Reply:
No, it does not need it. We needed to add llvm namespace because of lines 40-42 (which I removed). I opted to just include UniformityAnalysis.h

Reply:
The build failure says we do still need it :) I have just pushed a fix directly to your branch.

Reply:
I am sorry Jay ! You were right :)